### PR TITLE
Terraform code amended to correct CKV_TF_2 error.

### PR DESCRIPTION
**Pull request by SteveLinden**
## A reference to the issue / Description of it

As part of the check #6989 some errors are listed on checkov - CKV_TF_2 based on [ckv_ft_2 errors](https://github.com/ministryofjustice/modernisation-platform/actions/runs/9121756649) from the list of actions in [workflow action status]( https://user-guide.modernisation-platform.service.justice.gov.uk/user-guide/workflow-status.html#modernisation-platform-tools-amp-checks). The error relates to the code having no tag with a version number.

The code has been amended to put so that connects to the correct version of the called code and has removed the version number which is no longer needed.

Note, if terraform change the code, in the future, this will require updates to be made to the code.

### CODE REVIEW

**Review by ep-93 — DISMISSED** [review]
We could add it here - https://github.com/ministryofjustice/modernisation-platform/actions/runs/9107939964/workflow#L35


We have discussed there - https://moj.enterprise.slack.com/archives/C013RM6MFFW/p1715857932159299

File: terraform/templates/modernisation-platform-environments-isolated/vpc.tf [hunk 1]
```diff
@@ -1,7 +1,7 @@
 module "vpc" {
   #checkov:skip=CKV_TF_1:Module registry does not support commit hashes for versions
-  source  = "terraform-aws-modules/vpc/aws"
-  version = "~> 5.0"
+
+  source  = "github.com/terraform-aws-modules/vpc/aws?ref=25322b6b6be69db6cca7f167d7b0e5327156a595" # v5.8.1
```
Comment (ASTRobinson) [inline terraform/templates/modernisation-platform-environments-isolated/vpc.tf:4]: this should be: `source  = "github.com/terraform-aws-modules/terraform-aws-vpc?ref=25322b6b6be69db6cca7f167d7b0e5327156a595" # v5.8.1`

File: terraform/templates/modernisation-platform-environments-isolated/vpc.tf [hunk 2]
```diff
@@ -19,8 +19,8 @@ module "vpc" {
 
 module "vpc_endpoints" {
   #checkov:skip=CKV_TF_1:Module registry does not support commit hashes for versions
-  source  = "terraform-aws-modules/vpc/aws//modules/vpc-endpoints"
-  version = "~> 5.0"
+
+  source  = "github.com/terraform-aws-modules/vpc/aws//modules/vpc-endpoints?ref=25322b6b6be69db6cca7f167d7b0e5327156a595" # v5.8.1
```
Comment (ASTRobinson) [inline terraform/templates/modernisation-platform-environments-isolated/vpc.tf:23]: and this one: `source  = "github.com/terraform-aws-modules/terraform-aws-vpc//modules/vpc-endpoints?ref=25322b6b6be69db6cca7f167d7b0e5327156a595" # v5.8.1`

**Review by dms1981 — APPROVED** [review]

File: terraform/templates/modernisation-platform-environments-isolated/vpc.tf (hunk 1 above)
Comment (SteveLinden) [reply terraform/templates/modernisation-platform-environments-isolated/vpc.tf:4]: Missed this one. I will correct it

File: terraform/templates/modernisation-platform-environments-isolated/vpc.tf (hunk 2 above)
Comment (SteveLinden) [reply terraform/templates/modernisation-platform-environments-isolated/vpc.tf]: Also corrected

**Review by ASTRobinson — APPROVED** [review]

**Pull request merged**